### PR TITLE
Stopwatch: simplify counter reading

### PR DESCRIPTION
**Pull request by mattico**

### CODE REVIEW

File: src/timer/stopwatch.rs [hunk 1]
```diff
@@ -65,12 +65,7 @@ macro_rules! stopwatches {
                 }
 
                 pub fn now(&self) -> Instant {
-                    let _high = 0;
-                    $(
-                        let _high = self.tim.cnt.read().$cnt_h().bits() as u32;
-                    )*
-                    let low = self.tim.cnt.read().$cnt().bits() as u32;
-                    Instant(low | (_high << 16))
+                    Instant(self.tim.cnt.read().bits())
```
Comment (mattico) [inline src/timer/stopwatch.rs:68]: @dotcypress is there a reason this wasn't done this way originally?

Comment (dotcypress) [reply src/timer/stopwatch.rs:68]: Hmm, I don't recall why I did this.

**Review by dotcypress — CHANGES_REQUESTED** [review]
Did you check it on real hardware? If so I would be happy to merge this

**Comment by mattico** [comment]
Yeah, works fine on real hardware. Just wanted to make sure!

**Comment by dotcypress** [comment]
Thank you @mattico 👍

**Comment by mattico** [comment]
Appreciate the fast responses :)

**Pull request merged**